students/johnpharmd/lesson07/html_render.py

```diff
@@ -0,0 +1,50 @@
+class Element(object):
+    """renders an html element"""
+
+    tag = ''
+    indent = '    '
+
+    def __init__(self, content=''):
+        self.open_tag = '<' + self.tag + '>'
+        self.close_tag = '</' + self.tag + '>'
+        self.content = [content]
+
+    def append(self, new_string):
+        # self.content = '<br>'.join([self.content, new_string])
+        return self.content.append(new_string)
+
+    def render(self, file_out, cur_ind=''):
+        file_out.write(self.open_tag)
+        print('open_tag ==', self.open_tag)
+        print('self.content ==', self.content)
+        for l in self.content:
+            for s in l:
+                file_out.write(cur_ind + s + '\n')
+        file_out.write(self.close_tag)
+
+
+class Html(Element):
+    """renders html"""
```

Comment:
you should define class variable `tag = 'html'`

you don't the `__init__` in all of the subclasses, you just need to define `tag` class variable for each child class